[2025-08-18 13:15:45,377] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T11:15:37.816351+00:00 [queued]>
[2025-08-18 13:15:45,396] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T11:15:37.816351+00:00 [queued]>
[2025-08-18 13:15:45,397] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 13:15:45,397] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-18 13:15:45,397] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 13:15:45,417] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): verify_pipeline> on 2025-08-18 11:15:37.816351+00:00
[2025-08-18 13:15:45,424] {standard_task_runner.py:52} INFO - Started process 3872 to run task
[2025-08-18 13:15:45,429] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'verify_pipeline', 'scheduled__2025-08-18T11:15:37.816351+00:00', '--job-id', '281', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmpt2d44yjn', '--error-file', '/tmp/tmp3m8oeao4']
[2025-08-18 13:15:45,430] {standard_task_runner.py:80} INFO - Job 281: Subtask verify_pipeline
[2025-08-18 13:15:45,457] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-18 13:15:45,516] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T11:15:37.816351+00:00 [running]> on host a4b544519f2b
[2025-08-18 13:15:45,717] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=verify_pipeline
AIRFLOW_CTX_EXECUTION_DATE=2025-08-18T11:15:37.816351+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-18T11:15:37.816351+00:00
[2025-08-18 13:15:45,718] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-18 13:15:45,720] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            echo " Attente traitement Logstash..." && sleep 30 &&\n            curl -s "http://elasticsearch:9200/_cat/indices/weather*" || echo "Pas encore d\'index weather"\n        ']
[2025-08-18 13:15:45,752] {subprocess.py:85} INFO - Output:
[2025-08-18 13:15:45,768] {subprocess.py:92} INFO -  Attente traitement Logstash...
[2025-08-18 13:16:15,824] {subprocess.py:92} INFO - yellow open weather_data BQjbDtS5TvO1Q2HrAJGyvA 1 1 69 0 40.8kb 40.8kb
[2025-08-18 13:16:15,828] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-18 13:16:15,890] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=verify_pipeline, execution_date=20250818T111537, start_date=20250818T131545, end_date=20250818T131615
[2025-08-18 13:16:15,938] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 13:16:15,988] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
